docs/faq.rst

```diff
@@ -45,6 +45,23 @@ repository is only modified from one place. Also keep in mind that
 Borg will keep an exclusive lock on the repository while creating
 or deleting archives, which may make *simultaneous* backups fail.
 
+Can I back up to multiple, swapped backup targets?
+--------------------------------------------------
+
+To protect my backups against data loss by malware or catastrophes like a fire I
+want to back up to weekly swapped-out USB disks. Then I place the media
+currently not in use offline and off-site to keep them safe. The swapping out
+can be as simple as switching between two disks regularly, or in more elaborate
+schemes like
+`Grandfather-Father-Son <https://en.wikipedia.org/wiki/Backup_rotation_scheme>`_.
```

Comment:
somehow the reader expects an answer to the question in the headline here.

but this reads like a situation description of the user asking the question. i think this should be rephrased so it rather looks like an answer.

Reply:
Rephrased.